test/api/requests/auth.test.js

```diff
@@ -190,5 +191,37 @@ describe("Authentication request", () => {
         .get("/auth/github/callback?code=auth-code-123abc")
         .expect(401, done)
     })
+
+    it("should redirect to a redirect path if one is set in the session", done => {
+      const sessionKey = crypto.randomBytes(8).toString("hex")
+      const sessionBody = {
+        cookie: {
+          originalMaxAge: null,
+          expires: null,
+          httpOnly: true,
+          path: "/"
+        },
+        authRedirectPath: "/path/to/something",
```

Comment:
I get this string matches the string below in L#222, but can we make this a const value instead of trusting the two different string literals to match?